# archaea_project
Linux Bioinformatics Project

Methanogenesis is an anaerobic microbial metabolism that results in the production of methane, a potent greenhouse gas. The microbes responsible for this process are methanogenic Archaea (methanogens), which vary across different ecosystems and are significantly influenced by the pH of the environment. At Notre Dame, we were attempting to culture pH-resistant methanogenic Archaea from lake sediments to use in some experiments. From one enrichment, there were produced 50 different isolated colonies and the task was to identify which ones are the best candidates for growth experiments. Methanogens can be identified by the presence of the methyl-coenzyme M reductase (McrA) gene, which catalyzes the last step of methanogenesis and is highly conserved across all methanogens. pH resistance can be identified by the number of copies of the HSP70 gene, which is involved in protein biogenesis and refolding for stress resistance (Macario et al. 1999). You have been provided the proteomes (the translated genomes) of all 50 isolates in the proteomes directory, as well as reference protein sequences for the McrA gene and the HSP70 gene.

We used Unix code and other bioinformatics tools to determine which isolates the student should move forward with.

The code is provided in a single bash script.


